plugin/trino-druid/src/main/java/io/trino/plugin/druid/DruidJdbcClient.java

```diff
@@ -148,6 +150,9 @@ public ResultSet getTables(Connection connection, Optional<String> schemaName, O
                     return Optional.of(varcharColumnMapping(createUnboundedVarcharType(), true));
                 }
                 return Optional.of(defaultVarcharColumnMapping(columnSize, true));
+            case Types.TIMESTAMP:
+                // TODO: use {@link #timestampColumnMapping} when https://issues.apache.org/jira/browse/CALCITE-1630 gets resolved
```

Comment:
The Javadoc tags don't work in inline comments - change `{@link #timestampColumnMapping}` to `timestampColumnMapping`.

Reply:
the https://issues.apache.org/jira/browse/CALCITE-1630 issue is 4y old :/